data-raw/create_adams_data.R

```diff
@@ -122,18 +122,18 @@ run_template <- function(tp) {
 
 
 if (update_pkg) {
-  # github_pat <- Sys.getenv("GITHUB_TOKEN") # in case of run through github workflows
-  # if (github_pat == "") {
-  #   token_file <- "data-raw/github_token.R"
-  #   if (file.exists(token_file)) {
-  #     source(token_file)
-  #   } else {
-  #     "WARNING - you might have issues installing pharmaverse deps -
-  #           Please follow instructions in the readme.md"
-  #   }
-  # }
+  github_pat <- Sys.getenv("GITHUB_TOKEN") # in case of run through github workflows
```

Comment:
@kaz462 since we made `GITHUB_TOKEN` optional  (line 155) I can we can remove this part : 
```
  if (github_pat == "") {
    token_file <- "data-raw/github_token.R"
    if (file.exists(token_file)) {
      source(token_file)
    } else {
      "WARNING - you might have issues installing pharmaverse deps -
            Please follow instructions in the readme.md"
    }
  }
  ```

Reply:
removed. Any idea why the workflows have failed?

Reply:
yes global issue from admiralci, @cicdguy pushed a fix actually :) https://github.com/pharmaverse/admiralci/pull/190/files 
So we might just need to rerun once it's merged